debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/RecordMakers.java

```diff
@@ -126,6 +126,8 @@ protected RecordsForCollection(CollectionId collectionId, SourceInfo source, Str
             this.emitTombstonesOnDelete = emitTombstonesOnDelete;
         }
 
+
+
```

Comment:
Remove empty line changes. To minimise changes from base source..

Reply:
+1